client/components/webpack-build-monitor/index.jsx

```diff
@@ -111,7 +111,7 @@ class WebpackBuildMonitor extends React.Component {
 	componentDidMount() {
 		this.unsubscribe = store.subscribe( () => {
 			const status = store.getState();
-			if ( status !== this.state.status ) {
+			if ( ! isEqual( status, this.state ) ) {
```

Comment:
Whoops! `this.state.status` doesn't even exist after #17354, that was a bug 😅 

Could we just make this a `PureComponent` and skip checking altogether?

Reply:
@samouri Both `PureComponent` and `combineReducers` are smart and do the "right thing":

`PureComponent` [does a shallow comparison](https://facebook.github.io/react/docs/react-api.html#react.purecomponent) on both `props` and `state`.

`combineReducers` returns an unchanged previous state if none of subreducers returned a new state object.

That's why I thing that a `!==` comparison should be sufficient here. You can make the change in #17545 if you think it's a good idea.

Not doing the check would mean that `setState` is called more often than necessary -- on every action dispatch, even if it didn't change the state at all.

Reply:
PureComponent it is then 👍